package.json

```diff
@@ -21,15 +21,14 @@
     "test:cucumber": "cucumber-js -f @cucumber/pretty-formatter \"test/_features/**/*.feature\" --require test/_features/test.setup.js --require \"test/_features/step_definitions/**/*.js\" --tags $TAGS",
     "test:integration": "NODE_ENV=development _mocha \"test/_integration/**/*.spec.js\" -r dotenv/config --exit",
     "test:lint": "eslint . --config ./node_modules/eslint-config-hof/default.js",
-    "test:snyk": "snyk config set api=SNYK_TOKEN && snyk test",
     "build": "hof-build",
     "postinstall": "yarn run build"
   },
   "author": "HomeOffice",
   "dependencies": {
     "browserify": "^17.0.0",
     "deprecate": "^1.0.0",
-    "hof": "^20.4.0",
+    "hof": "^21.0.6",
```

Comment:
Could you use `~21.0.6` here please? Hof should now be updated to the latest _patch_ version in package.json